draft-ietf-dnsop-svcb-https.md

```diff
@@ -244,7 +244,7 @@ The SVCB DNS resource record (RR) type (RR type 64)
 is used to locate alternative endpoints for a service.
 
 The algorithm for resolving SVCB records and associated
-address records is specified in {{client-behavior}}.
+address records are specified in {{client-behavior}}.
```

Comment:
Nope, singular :).  "The algorithm ... is specified".  We can rephrase if that sounds weird.

Reply:
Arrfg - you are right.  I did miss this